filters/includes.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python
+
+"""
+Process a CodeBlock and, if class is "include", search for the file in the
+provided paths, and replace the CodeBlock with the (formatted) file contents.
```

Comment:
aside: i'm not sure if saying (formatted) is necessary here. it implies that pandoc returns formatted elements, (if i recall correctly)